server/models/GameLogic.js

```diff
@@ -361,9 +361,9 @@ class GameLogic {
 
     updateLabels(word) {
         this.playedCards.forEach(card => {
-            const cardScore = this.votes.filter(vote => card.cardId === vote.cardId).length
+            const cardScore = this.votes.filter(vote => card.cardId === vote.cardId && !vote.username.includes("CPU")).length;
```

Comment:
Someone may have CPU in their username, is it better to find the user and use the `user.real` flag